core/iwasm/common/wasm_runtime_common.c

```diff
@@ -863,6 +863,183 @@ wasm_runtime_call_wasm(WASMExecEnv *exec_env,
     return false;
 }
 
+uint32 parse_args_to_uint32_array(WASMType *type,
```

Comment:
Define as static uint32, and let return type in a single line:
static uint32
parse_args_to_uint32_array(...